tests/e2e/client.spec.ts

```diff
@@ -75,6 +76,16 @@ describe("DuneClient: native routes", () => {
     expect(execution.execution_id).is.not.null;
   });
 
+  it("execute with Large tier performance", async () => {
+    const client = new DuneClient(apiKey);
+    const execution = await client.executeQuery(
+      1215383,
+      undefined,
+      ExecutionPerformance.Large,
+    );
```

Comment:
This makes me think that we need to define a type for the parameters. So we don't have to insert undefined in between two optional ones.